lua/pantran/curl.lua

```diff
@@ -5,8 +5,13 @@ local curl = {
   config = {
     cmd = "curl",
     retry = 3,
-    timeout = 15
-  }
+    timeout = 15,
+      proxy = {
+        enable = false,
```

Comment:
Why do we need a separate `enable` field? Can't we just use `vim.tbl_isempty` on the entire table?